modules/local/virusbreakend/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: virusbreakend
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bioconda::gridss=2.13.2=h50ea8bc_3
```

Comment:
same here for the build number

Reply:
See https://github.com/nf-core/oncoanalyser/pull/29#discussion_r1697983262